wolfssl/wolfcrypt/asn.h

```diff
@@ -811,6 +811,44 @@ extern const WOLFSSL_ObjectInfo wolfssl_object_info[];
 
 #define WC_NID_undef 0
 
+/* Setup for WC_MAX_RSA_BITS needs to be here, rather than rsa.h, because
+ * FIPS headers don't have it.  And it needs to be here, rather than internal.h,
+ * so that setup occurs even in cryptonly builds.
+ */
+#ifndef NO_RSA
+    #ifndef WC_MAX_RSA_BITS
+        #ifdef USE_FAST_MATH
+            /* FP implementation support numbers up to FP_MAX_BITS / 2 bits. */
+            #define WC_MAX_RSA_BITS    (FP_MAX_BITS / 2)
```

Comment:
Not a good place for this. asn.h is private.
Try types.h

Reply:
we're moving this from `internal.h` -- it was already private, and until we have a reason to make it non-private we should keep it private.

Reply:
oh and also, we can't put the setup for `WC_MAX_RSA_BITS`  in `types.h` or `settings.h` because it depends on `tfm.h` or `sp_int.h`, and since `WC_MAX_CERT_VERIFY_SZ` depends on `WC_MAX_RSA_BITS`, we can't put setup for `WC_MAX_CERT_VERIFY_SZ` in `types.h` or `settings.h` either.

the setup for this is very finicky, I discovered during dev -- if it's defined wrong, the result is buffer overruns.  lost quite a few cycles chasing that down...